spec/openapi.yaml

```diff
@@ -0,0 +1,271 @@
+openapi: 3.0.1
+info:
+  title: PrusaLink Web
+  description:
+    This is a printer webservice.
+  contact:
+    email: link@prusa3d.cz
+  license:
+    name: AGPLv3
+  version: 1.0.0-draft
+externalDocs:
+  description: Find out more about API
+  url: http://connect.prusa3d.cz
+security:
+  - digestAuth: []
+paths:
+  /api/version:
+    get:
+      summary: api version informations
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Version"
+
+  /api/v1/info:
+    get:
+      summary: printer informations
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Info"
+        401:
+          $ref: "#/components/responses/Unauthorized"
+
+components:
+  schemas:
+    Version:
+      type: object
+      properties:
+        api:
+          type: string
+          example: "1.0.0"
+        version:
+          type: string
+          example: "0.7.0"
+        printer:
+          type: string
+          example: "1.3.1"
+        text:
+          type: string
+          example: "PrusaLink 0.7.0"
```

Comment:
What exactly should the there be?

Reply:
As example say:
* PrusaLink 0.7.0
* Original Prusa MINI 4.3.1
* etc